ui/components/MessageBox.tsx

```diff
@@ -3,7 +3,7 @@ import Flex from "./Flex";
 
 const MessageBox = styled(Flex)`
   box-sizing: border-box;
-  width: 560px;
+  width: 50%;
```

Comment:
A `max-width` may can help here with non-standard screen sizes, like someone on 4k monitor (not me), the box will be really big with 50%